### PR TITLE
[ACS-5347] New styles for facets and filters states

### PR DESCRIPTION
**Pull request by MichalKinas**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

UI and styles for facets and filters has been updated. https://alfresco.atlassian.net/browse/ACS-5347 https://alfresco.atlassian.net/browse/ACS-5360

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by DenysVuika — CHANGES_REQUESTED** [review]

File: lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.spec.ts [hunk 1]
```diff
@@ -65,4 +65,16 @@ describe('SearchWidgetChipComponent', () => {
       applyButton.triggerEventHandler('click', {});
       expect(queryBuilder.update).toHaveBeenCalled();
   });
+
+  it('should display arrow down icon',  () => {
+    const icon = fixture.debugElement.query(By.css('mat-chip mat-icon')).nativeElement.innerText;
+    expect(icon).toEqual('keyboard_arrow_down');
+});
```
Comment (AleksanderSklorz) [inline lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.spec.ts:72]: Probably this line and 79th are not aligned properly, at least it looks that here

File: lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.spec.ts [hunk 2]
```diff
@@ -65,4 +65,16 @@ describe('SearchWidgetChipComponent', () => {
       applyButton.triggerEventHandler('click', {});
       expect(queryBuilder.update).toHaveBeenCalled();
   });
+
+  it('should display arrow down icon',  () => {
```
Comment (AleksanderSklorz) [inline lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.spec.ts:69]: double space

File: lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.spec.ts [hunk 3]
```diff
@@ -65,4 +65,16 @@ describe('SearchWidgetChipComponent', () => {
...
+    const icon = fixture.debugElement.query(By.css('mat-chip mat-icon')).nativeElement.innerText;
+    expect(icon).toEqual('keyboard_arrow_down');
+});
+
+it('should display arrow up icon when menu is opened',  () => {
```
Comment (AleksanderSklorz) [inline lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.spec.ts:74]: double space

File: lib/content-services/src/lib/search/components/search-filter-chips/search-facet-chip/search-facet-chip.component.scss [hunk 4]
```diff
@@ -1,5 +1,10 @@
 .adf-search-filter-chip {
     &[disabled] {
         pointer-events: none;
+
+        &.mat-chip {
+            background-color: #f5f5f5;
```
[the remark this answers is not in the document]
Comment (MichalKinas) [reply lib/content-services/src/lib/search/components/search-filter-chips/search-facet-chip/search-facet-chip.component.scss]: Good point, I realized that these colors are ACA specific styles so I moved them to the ACA PR

File: lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.spec.ts (hunk 1 above)
Comment (MichalKinas) [reply lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.spec.ts]: You're right, couple of places were misaligned, fixed now

File: lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.spec.ts (hunk 3 above)
Comment (MichalKinas) [reply lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.spec.ts]: Removed

**Review by DenysVuika — APPROVED** [review]

**Pull request merged**